spec/models/child_spec.rb

```diff
@@ -984,6 +984,47 @@
     end
   end
 
+  describe "#find_in_form" do
```

Comment:
We're gonna need a whole lot more tests here. I am guessing a lot of the stuff you put into the fixtures can be turned into tests.